src/mito2/src/region/options.rs

```diff
@@ -163,6 +172,7 @@ impl Default for TwcsOptions {
             max_active_window_files: 4,
             max_inactive_window_files: 1,
             time_window: None,
+            remote_compaction: false,
```

Comment:
Also update the tests in this file to see whether we can deserialize the `remote_compaction` flag correctly.